### PR TITLE
Add error message for login page

### PR DESCRIPTION
**Pull request by gingerbreadpup**
I was going to add an error message for each login field, but after checking the OWASP docs and also seeing how it was done on MOMO, I decided to just have one error message display. Rather than telling the user which field was incorrect, it just says that the username and password combination they entered wasn't recognised.

### CODE REVIEW

File: src/pages/Login.vue [hunk 1]
```diff
@@ -7,15 +7,15 @@
 
                 <h1>Log In</h1>
 
+                <div class="message">The username and password combination entered was not recognised. Please check your details and try again.</div>
```
Comment (katjam) [inline src/pages/Login.vue]: Fine to start but we probably need these to be the messages returned by the API. Are there any there?

File: src/pages/Login.vue [hunk 2]
```diff
@@ -28,17 +28,21 @@
...
-
     export default {
         name: 'login',
         data: function () {
             return {
                 username: null,
                 password: null,
-                remember: true
+                remember: true,
+                errorMessage : Store.errors[0]
```
Comment (charlesstrange2) [inline src/pages/Login.vue]: oddly, i dont think that [0] is necessary here. unless you *really are* watching the first error message in the array?

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
see if you really need it to be store.errors[0]

**Review by charlesstrange2 — APPROVED** [review]
:sheep::it:

**Pull request merged**